packages/design-system-docs/src/pages/components/Tooltip/Tooltip.example.jsx

```diff
@@ -5,90 +5,97 @@ import ReactDOM from 'react-dom';
 ReactDOM.render(
   <>
     <div className="ds-u-display--flex ds-u-align-items--center ds-u-margin-y--2">
-      <p className="ds-u-margin--0">Tooltip using an icon trigger</p>
+      <p className="ds-u-margin--0">Tooltip with icon trigger</p>
       <Tooltip
         ariaLabel="Label describing the subject of the tooltip"
-        triggerContent={<TooltipIcon />}
-        triggerClassName="ds-c-tooltip__trigger-icon"
-        triggerActiveClassName="ds-c-tooltip-icon--active"
+        className="ds-c-tooltip__trigger-icon"
```

Comment:
@LouisFettet Just so you are aware, I did end up deciding to swap how the user specifies the tooltip trigger and content. Now the `children` corresponds to the `trigger`, and `title` corresponds to the tooltip content.

This allows developers to have more flexibility with customizing the trigger content. It also means additional props will be able to automatically passed through to the trigger element.

Reply:
Ah I see, I was trying to follow the same naming convention as native HTML tooltips, which use the `title` attribute. Its a bit tricky, because the native HTML tooltip is really quite bad, but other design systems like [USWDS](https://designsystem.digital.gov/components/tooltip/) also seem to follow that naming convention

Reply:
Gotcha, that makes more sense in context. I'll leave this up to your best judgment then!